### PR TITLE
safe harbor statement

### PR DESCRIPTION
**Pull request by lukaselmer**

### CODE REVIEW

**Comment by lukaselmer** [comment]
please merge if you like it, and create a pr from develop to master, thx!

File: src/documents/news/2015-10-20-safe-harbor-statement.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: Safe-Harbor-Urteil des Europäischen Gerichtshofs: Stellungnahme der Renuo GmbH
+date: 2015-10-20
+image_path: /images/news/safe-harbor-not-anymore.jpg
```
Comment (diegosteiner) [inline src/documents/news/2015-10-20-safe-harbor-statement.html]: are we allowed to use that image?

File: src/documents/news/2015-10-20-safe-harbor-statement.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+immer möglich werden Nutzerdaten aber in der Schweiz oder in der EU gespeichert, damit der Datenschutz gewährleistet ist. Allerdings macht
+es auch Sinn, gewisse Daten (zum Beispiel verschlüsselte Datenbank Backups) an physisch anderen Orten zu speichern, damit die Verfügbarkeit
+gewährleistet ist (zum Beispiel bei einem Ausfall eines Rechenzentrums).</p>
+
+<p>2000 hat die Europäische Kommission das sog. bilateral abgesprochene "Safe-Harbor-Abkommen" eingeführt,  aufgrund dessen es Unternehmen
+ermöglicht werden sollte, in Übereinstimmung mit europäischen Datenschutzrichtlinien Daten aus einem Land der Europäischen Union in die USA
+zu übermitteln. Am 6. Oktober 2015 hat der Europäische Gerichtshof (EuGH) das Abkommen für ungültig erklärt.</p>
+
+<p>Die Schweiz hat, unabhängig vom ursprünglichen Abkommen, ein eigenes ausgehandeltes Abkommen mit den USA (
+<a href="http://www.edoeb.admin.ch/dokumentation/00153/00262/00278/index.html?lang=de"> U.S.-Swiss Safe Harbor Framework</a>). Dieses wurde
```
Comment (diegosteiner) [inline src/documents/news/2015-10-20-safe-harbor-statement.html]: IMHO a space too much right after the opening parenthesis ( U.S.-Swiss Safe...)

File: src/documents/news/2015-10-20-safe-harbor-statement.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+nicht gekündigt und ist weiterhin gültig. Durch die Ungültigkeitserklärung ist die Zukunft jedoch ungewiss. Insofern sind derzeit nur Kunden
```
Comment (diegosteiner) [inline src/documents/news/2015-10-20-safe-harbor-statement.html]: "nicht gekündigt" emphasize or strong this?

File: src/documents/news/2015-10-20-safe-harbor-statement.html (hunk 2 above)
Comment (lukaselmer) [inline src/documents/news/2015-10-20-safe-harbor-statement.html]: :+1:

File: src/documents/news/2015-10-20-safe-harbor-statement.html (hunk 1 above)
Comment (lukaselmer) [inline src/documents/news/2015-10-20-safe-harbor-statement.html]: :+1:

File: src/documents/news/2015-10-20-safe-harbor-statement.html (hunk 3 above)
Comment (lukaselmer) [inline src/documents/news/2015-10-20-safe-harbor-statement.html]: :+1:

**Comment by lukaselmer** [comment]
done, thx!

File: src/documents/news/2015-10-20-safe-harbor-statement.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+<a href="https://de.wikipedia.org/wiki/Hypertext_Transfer_Protocol_Secure">HTTPS</a>). Dadurch können Unberechtigte die Leitung zwischen
+Ihren Kunden und dem Server nicht abhören oder verändern.</p>
+
+<p>PPS: Die Diskussion rund um das Thema kann unseres Erachtens stark ad absurdum geführt werden. Wann ist zum Beispiel der Moment, in
+welchem Kundendaten übermittelt werden? Genügt es bereits, wenn ein Endkunde (Wohnsitz Deutschland) in den Ferien in den USA einen Artikel
+aus Ihrem Online Shop bestellt? Oder sich während seinen USA-Ferien in sein Benutzerkonto einloggt? Unabhängig des Orts des Hostings können
+so zum Beispiel Kundendaten ins Ausland gelangen. Um 100% sicher zu gehen müsste man also den Zugriff auf Ihre Webseite von ausserhalb
+Europa sperren? Unserer Meinung nach ginge dies zu weit. Unsere Interpretation des Themas geht soweit, als dass wir den Ort des Hostings der
+personenbezogenen Daten als relevant betrachten. Und diese sind bei unseren Produkten in den meisten Fällen innerhalb von Europa (meistens
+Schweiz, Frankfurt oder Irland).</p>
```
Comment (schmijos) [inline src/documents/news/2015-10-20-safe-harbor-statement.html]: Frankfurt -> Deutschland

Comment (lukaselmer) [inline src/documents/news/2015-10-20-safe-harbor-statement.html]: Good point! :+1:

File: src/documents/news/2015-10-20-safe-harbor-statement.html (hunk 1 above)
Comment (ideadapt) [inline src/documents/news/2015-10-20-safe-harbor-statement.html]: hi jungs. wänni das so xehn chunt mr spontan das do inda sinn: www.nachrichtendienstgesetzt.ch - wär wänner üs unterstützend :dancer:

Comment (lukaselmer) [inline src/documents/news/2015-10-20-safe-harbor-statement.html]: hello! :smile: sprich das gern mal bime :coffee: a, thx :+1:

**Pull request merged**